### PR TITLE
David macdonald patch drag pointer

### PR DESCRIPTION
**Pull request by DavidMacDonald**
This patch addresses Issue #1225 and disambiguates dragging and path based gestures

### CODE REVIEW

File: understanding/21/pointer-gestures.html [hunk 1]
```diff
@@ -20,14 +20,16 @@ <h2>Intent of this Success Criterion</h2>
...
+     
+      <h3>Disambiguation of Pointer Gestures and Dragging</h3>
 
-			<p>Dragging motions where the movement between the start and end points can go in any direction and is entirely non-prescriptive are not path-based. Dragging motions are covered in Success Criterion 2.5.7 Dragging.</p>
+      <p>Dragging is a motion between the start and end points without an intermediate point which can go in any direction and is entirely non-prescriptive, and therefore not path-based.  Although a user could drag in the same way as a gesture, the difference is that a gesture requires an intermediate point, dragging does not. Path-based gestures are a communication of a particular command to the system. Dragging involves the displacement of an object from one location to another, gestures do not.  Dragging motions are covered in <a href="https://www.w3.org/WAI/WCAG22/Understanding/dragging.html">Success Criterion 2.5.7: Dragging</a>.</p>
```
Comment (detlevhfischer) [inline understanding/21/pointer-gestures.html:26]: For consistency#s sake we should either use "dragging motion" or "dragging movement" in both 2.5.1 and 2.5.7. I leave it to native speakers to determine the better choice.
I believe this text could ans should be shorter (maybe not quire as short as the one it replaces). I can try to rephrase...

File: understanding/21/pointer-gestures.html [hunk 2]
```diff
@@ -20,14 +20,16 @@ <h2>Intent of this Success Criterion</h2>
 			</figure>
 
 			<p>Examples of path-based gestures include swiping, sliders and carousels dependent on the direction of interaction, and other gestures which trace a prescribed path such as drawing a specific shape. Such paths may be drawn with a finger or stylus on a touchscreen, graphics tablet, or trackpad, or with a mouse, joystick, or similar pointer device.</p>
+     
+      <h3>Disambiguation of Pointer Gestures and Dragging</h3>
```
Comment (detlevhfischer) [inline understanding/21/pointer-gestures.html:24]: A simpler heading (think COGA) would be "The difference between pointer gestures and dragging"

File: understanding/22/dragging-movements.html [hunk 3]
```diff
@@ -48,8 +48,10 @@ <h2>Intent of Dragging Movements</h2>
...
+      <p>In contrast to dragging movements, path-based gestures involve at least an initial directionality of the pointer movement. This is defined by stating that the path has to go through at least one intermediate point to qualify as pointer gesture. The intermediate point defines the gesture as requiring a specific path, even if the complete path is not defined. So, although a user could drag in the same way as a gesture, the difference is that a gesture requires an intermediate point, dragging does not. For more details, refer to <a href="https://www.w3.org/WAI/WCAG21/Understanding/pointer-gestures.html">Understanding Success Criterion 2.5.1 Pointer Gestures</a>.</p>
```
Comment (detlevhfischer) [inline understanding/22/dragging-movements.html:52]: In my view, the sentences added: "The intermediate point defines the gesture as requiring a specific path, even if the complete path is not defined. So, although a user could drag in the same way as a gesture, the difference is that a gesture requires an intermediate point, dragging does not" make the text more verbose without providing more clarity. I would leave the text as it is.

**Review by detlevhfischer — CHANGES_REQUESTED** [review]
See my PR #1638 for what I believe is a more understandable explanation of the difference between path-based gestures and dragging.

**Comment by alastc** [comment]
Agreed in the meeting, although further updates are likely.

**Pull request merged**